org.contextmapper.dsl/src/org/contextmapper/dsl/ContextMappingDSL.xtext

```diff
@@ -60,7 +60,9 @@ BoundedContext:
 			('knowledgeLevel' ('=')? knowledgeLevel=KnowledgeLevel)?)
 			((application = Application)? &
 			(modules += SculptorModule)* &
-			(aggregates += Aggregate)*)
+			(aggregates += Aggregate)* &
+            ('businessModel' ('=')? businessModel=BusinessModel)? &
```

Comment:
I would move these attributes up (below `knowledgeLevel`) so that we have the single attributes together. And then the aggregates and modules at the end as before.